.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+nodejs:
+  - node
+services:
+  - postgresql
+
+install:
+  - npm install
+
+script:
+  # - npm run test
+
+notifications:
+  slack: andela:E6PvFgt7ygC8QJHxXQJ7mEML
+  
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
+
+after_script:
+ # -run after all script is executed successfully
+
+ after_success:
+  # - run all jobs is successfull
+
+after_failure:
+  # - run if any job fails
```

Comment:
A new line needed here. You can add a .editorconfig file to enforce this in all your files

Reply:
@benfluleck this has been fixed